docs/proposals/proposal-001-trigger-and-deploy.md

```diff
@@ -116,33 +116,38 @@ removed. In future we could create nodes on demand and delete on completion.
 
 ### Subscribing to Releases
 
-A YAML file of CNCF projects and any sub components will be stored in the
+A JSON file of CNCF projects and any sub components will be stored in the
 tooling repo e.g.
 
-```yaml
-# projects.yaml
-projects:
-  - name: falco
-    sub_components:
-      - ebpf
-      - modern-ebpf
-      - kmod
+```json
+{
+    "projects": [
+        {
+            "name": "falco",
+            "organization": "falcosecurity",
+            "sub_components": [
```

Comment:
We discussed in the meeting that we could rename this to `configurations` rather than `sub_components`

Reply:
@nikimanoledaki Yes, I think configurations is a much better name.

Proposal updated to reflect this and the workflow input is renamed from `cncf_project_sub` to `config`.